schema.json

```diff
@@ -2173,7 +2173,7 @@
           {
             "group": "Montserrat",
             "label": "Montserrat",
-            "value": "Google_Montserrat_400"
+            "value": "Google_Montserrat_600"
```

Comment:
I understand that we are changing the default from 400 to 600. But I think we should keep 400 in the list to be backward compatible ?

Reply:
what do you mean with backward compatible? Merchant would have to opt-in to the new update. Also, this is only changing `checkout` fonts, not main storefront fonts.

Also merchants don't have a way to change font weight in theme editor.

Reply:
When a merchant updates their theme their configuration changes are merged but their saved values are preserved. So when the merchant uses theme editor to switch font he won’t have access to his previous value since it won’t be available in the option set.

Reply:
right, I see 👁  would probably need to test this scenarios to see how it works

Reply:
Added `Google_Montserrat_400` as `Montserrat Light` for backward compatibility.